Kickstarter-iOS/Library/SharedFunctions.swift

```diff
@@ -47,3 +47,15 @@ public func logoutAndDismiss(
 
   viewController.dismiss(animated: true, completion: nil)
 }
+
+// MARK: - Email Verification workflow
+
+public func startEmailVerificationViewController(
```

Comment:
I'm guessing this is more likely to be presented modally so we might name this
```swift
public func presentEmailVerificationViewController(on viewController: UIViewController) { ... }
```
Also a little confusing but there are two `SharedFunctions.swift` one in the `Library` framework and one in `Kickstarter_Framework`. I think this is better suited to the latter because it'll be called from VC's and not view models.

Reply:
According to the designs, it looks like a push (back arrow on top left of view controller). The only thing I find weird is the interaction with going "back" after logging in or signing up...

Reply:
oh, interesting - i think we should just confirm that behaviour (when it will be pushed onto the stack and when it will be modal)

Reply:
brought this up in the email-verification channel for confirmation/discussion